packages/dcos-integration-test/extra/util/delete_ec2_volume.py

```diff
@@ -88,7 +88,7 @@ def _delete_volume(volume):
         try:
             log.info("Issuing volume.delete()")
             volume.delete()  # Raises ClientError (VolumeInUse) if the volume is still attached.
-        except exceptions.ClientError as exc:
```

Comment:
My local linter picked this unused variable up so I removed it.

Reply:
Please as a separate commit next time.